quickstart/app/Models/Task.php

```diff
@@ -8,9 +8,12 @@
 class Task extends Model
 {
     use HasFactory;
+    protected $fillable = [
+        'name',
+    ];
 
     public function user()
     {
-        return $this->belongsTo(User::class, 'user_id','id');
+        return $this->belongsTo(User::class,'user_id','id');
```

Comment:
```suggestion
        return $this->belongsTo(User::class, 'user_id', 'id');
```
thêm khoảng trắng sau dấu phẩy nhé.